functions/AutoRemediateEC2-003.js

```diff
@@ -21,11 +20,11 @@ module.exports.handler = (event, context, callback) => {
       return handleError(err.message ? err.message : 'removing RDP database access on port RDP failed')
     }
     console.log('Result', result)
-    return callback(null, 'Successfully processed event')
+    return 'Successfully processed event'
```

Comment:
🔧 This needs to use `callback` because this is inside of a `callback` of the `AccessRevoker.revoke` method.

Same for the`handleError` method below